### PR TITLE
feature: cmp support new import method

### PR DESCRIPTION
**Pull request by iutx**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:
cmp: support new import method
#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @luobily 

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/cmp/endpoints/cluster.go [hunk 1]
```diff
@@ -409,3 +420,127 @@ func (e *Endpoints) validateOpsConfig(opsConf *apistructs.OpsConfig) error {
...
+		errStr := fmt.Sprintf("failed to unmarshal to apistructs.ImportCluster: %v", err)
+		return mkResponse(apistructs.ImportClusterResponse{
+			Header: apistructs.Header{
+				Success: false,
+				Error:   apistructs.ErrorResponse{Msg: errStr},
+			},
+		})
+	}
+
+	logrus.Debugf("cluster inti retry reuqest body: %v", req)
```
Comment (luobily) [inline modules/cmp/endpoints/cluster.go:471]:
```suggestion
	logrus.Debugf("cluster init retry reuqest body: %v", req)
```

File: modules/cmp/endpoints/cluster.go [hunk 2]
```diff
@@ -409,3 +420,127 @@ func (e *Endpoints) validateOpsConfig(opsConf *apistructs.OpsConfig) error {
...
```
Comment (luobily) [inline modules/cmp/endpoints/cluster.go:437]:
```suggestion
	logrus.Debugf("cluster init retry reuqest body: %v", req)
```

File: modules/cmp/impl/clusters/proxy-deploy.go [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  namespace: erda-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: erda-crb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```
Comment (luobily) [inline modules/cmp/impl/clusters/proxy-deploy.go:38]: Where will this role be created?

Comment (iutx) [reply modules/cmp/impl/clusters/proxy-deploy.go:38]: cluster-admin is a default kubernetes clusterrole, e.g.
```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  annotations:
    rbac.authorization.kubernetes.io/autoupdate: "true"
  creationTimestamp: "2021-04-14T11:34:01Z"
  labels:
    kubernetes.io/bootstrapping: rbac-defaults
  name: cluster-admin
  resourceVersion: "377509138"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterroles/cluster-admin
  uid: 4f2eebd5-2954-4016-b7af-d2054c719464
rules:
- apiGroups:
  - '*'
  resources:
  - '*'
  verbs:
  - '*'
- nonResourceURLs:
  - '*'
  verbs:
  - '*'
```

**Pull request merged**